packages/alert/src/index.vue

```diff
@@ -8,7 +8,7 @@
     >
       <i v-if="showIcon" class="el-alert__icon" :class="[ iconClass, isBigIcon ]"></i>
       <div class="el-alert__content">
-        <span v-if="title || $slots.title" class="el-alert__title" :class="[ isBoldTitle ]">
+        <span v-if="!!title || $slots.title" class="el-alert__title" :class="[ isBoldTitle ]">
           <slot name="title">{{ title }}</slot>
         </span>
         <p v-if="$slots.default || !!description" class="el-alert__description">
```

Comment:
@JeremyWuuuuu just for keep the same with this 😂，although， i think title || $slots.title is ok for me.

Reply:
Maybe remove the `!!` part on description in this case should be more reasonable to me though. We need another input on this. cc @HerringtonDarkholme @iamkun

Reply:
IMHO double negation does not make difference in this case, I think?

Reply:
Yeah, I think the double negation operation is something like "gild the lily" here for me. maybe @cbbfcd you can remove the double negation at line#14 for keeping consistency?

Reply:
In some company's specifications, double negation is the best case. I remember airbnb is like this?

Reply:
done.